scanners/zap/tests/test_setup_podman.py

```diff
@@ -171,6 +171,26 @@ def test_setup_ajax(test_config):
             break
 
 
+def test_setup_graphql(test_config):
+    TEST_GRAPHQL_ENDPOINT = "http://test.com/graphql"
+    TEST_GRAPHQL_SCHEMA_URL = "http://test.com/schema.graphql"
+    TEST_GRAPHQL_SCHEMA_FILE = "schema.graphql"
+
+    test_config.set("scanners.zap.graphql.endpoint", TEST_GRAPHQL_ENDPOINT)
+    test_config.set("scanners.zap.graphql.schemaUrl", TEST_GRAPHQL_SCHEMA_URL)
+    test_config.set("scanners.zap.graphql.schemaFile", TEST_GRAPHQL_SCHEMA_FILE)
+
+    test_zap = ZapPodman(config=test_config)
+    test_zap.setup()
+
+    for item in test_zap.af["jobs"]:
+        if item["type"] == "graphql":
+            assert item["parameters"]["endpoint"] == TEST_GRAPHQL_ENDPOINT
+            assert item["parameters"]["schemaUrl"] == TEST_GRAPHQL_SCHEMA_URL
+            assert item["parameters"]["schemaFile"] == TEST_GRAPHQL_SCHEMA_FILE
```

Comment:
if the suggestion made below are fine, then this should work:

```
assert item["parameters"]["schemaFile"] == f"{test_zap._host_work_dir()}/graphql.schema"
```
We could even make sure that the file was correctly copied with the following assertion:
```
assert Path(test_zap._host_work_dir(), "graphql.schema").is_file()
```
(and we could even check if the content of the file has not been modified, but this might be a little too much?)

Reply:
for the test to pass, I think it should be `test_zap._container_work_dir()` instead of `_host_work_dir()`?

Reply:
👍 Yes you are correct, we're testing the AF configuration, so it's the view from the container's perspective, `test_zap._container_work_dir()`:
```
assert item["parameters"]["schemaFile"] == f"{test_zap._container_work_dir()}/graphql.schema"

```

Reply:
(But the `assert Path(test_zap._host_work_dir(), "graphql.schema").is_file()`  line to test the actual file's existence should be correct)